examples/failure-notification/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: failure-notification-example
+spec:
```

Comment:
Need to add annotations aka (this is Matthew, adjust to your own for name a, URL and path)

```yaml
  annotations:
    argo-hub/version: '0.0.1'
    argo-hub/description: 'Workflow template for a full deployment using jira, sonar and a canary rollout'
    argo-hub/categories: 'argo'
    argo-hub/license: 'MIT'
    argo-hub/owner_name: 'Matthew Chung'
    argo-hub/owner_email: 'matthew.chung@codefresh.io'
    argo-hub/owner_avatar: 'https://github.com/matthewchungcodefresh.png'
    argo-hub/owner_url: 'https://github.com/matthewchungcodefresh'
    argo-hub/icon_url: "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/jira-sonar/assets/icon.svg"
    argo-hub/icon_background: "#f4f4f4"
```